blocks/header/header.css

```diff
@@ -494,3 +491,14 @@ header .nav-search-input .search_autocomplete .popover-container {
   margin-top: 20px;
   z-index: 3;
 }
+
+ul li.nav-drop > ul > li > ul {
+    margin: 1em 0 0 .5em;
+}
+
+@media (width < 900px) {
+    header nav .nav-sections ul > li > ul,
+    header nav .nav-sections ul > li > ul > li {
+        width: 100%;
+    }
```

Comment:
I am sure we need more styling for mobile using media queries.
it looks poor as it looks at the moment...
<img width="650" alt="Screenshot 2025-02-10 at 12 31 07" src="https://github.com/user-attachments/assets/e2f02d3c-d6d5-4fff-9d50-f1909c97264b" />
<img width="527" alt="Screenshot 2025-02-10 at 12 31 21" src="https://github.com/user-attachments/assets/c43227ab-2d5a-4280-84c9-b950f2b1ddc1" />